src/Compiler/Scheme/Racket.idr

```diff
@@ -326,18 +326,27 @@ startRacket : String -> String -> String -> String
 startRacket racket appdir target = unlines
     [ "#!/bin/sh"
     , ""
-    , "case `uname -s` in            "
+    , "set -e # exit on any error"
+    , ""
+    , "case $(uname -s) in           "
     , "    OpenBSD|FreeBSD|NetBSD)   "
-    , "        DIR=\"`grealpath $0`\""
+    , "        REALPATH=\"grealpath\""
     , "        ;;                    "
     , "                              "
     , "    *)                        "
-    , "        DIR=\"`realpath $0`\" "
+    , "        REALPATH=\"realpath\" "
     , "        ;;                    "
     , "esac                          "
     , ""
-    , "export LD_LIBRARY_PATH=\"$LD_LIBRARY_PATH:`dirname \"$DIR\"`/\"" ++ appdir ++ "\"\""
-    , racket ++ "\"`dirname \"$DIR\"`\"/\"" ++ target ++ "\" \"$@\""
+    , "if ! command -v \"$REALPATH\" >/dev/null                     "
+    , "then                                                         "
+    , "    echo \"$REALPATH is required for Racket code generator.\""
+    , "    exit 1                                                   "
+    , "fi                                                           "
+    , ""
+    , "DIR=$($REALPATH \"$0\")"
+    , "export LD_LIBRARY_PATH=\"$(dirname \"$DIR\")/\"" ++ appdir ++ "\":$LD_LIBRARY_PATH\""
+    , racket ++ "\"$(dirname \"$DIR\")\"/\"" ++ target ++ "\" \"$@\""
```

Comment:
```suggestion
    , "DIR=$(dirname \"$($REALPATH \"$0\")\")"
    , "export LD_LIBRARY_PATH=$DIR/" ++ appdir ++ ":$LD_LIBRARY_PATH"
    , racket ++ "\"$DIR/" ++ target ++ "\" \"$@\""
```
This removes a few superfluous quotes too. (Unless `appdir` and `target` are already shell quoted?)
BTW, you've changed to "Exit" in existing comments but not in new ones.

---

To reviewers: note the change in order, as was done in #709 for Chez.

Reply:
Thanks, I applied your suggestions but I left `appdir` and `target` quoted just in case.

> BTW, you've changed to "Exit" in existing comments but not in new ones.

Could you point out the line where this is a case?